example/src/example.rs

```diff
@@ -12,3 +12,8 @@ fn main() {
         vprintf(b"%d %d %s %f\n\0".as_ptr() as *const c_char, v);
     }, 1, 2, b"salut!\0".as_ptr(), 32f32 as c_double);
 }
+
+#[cfg(target_os = "windows")]
+mod platform {
+	#[link(name = "legacy_stdio_definitions.lib")] extern {}
```

Comment:
In the tests it seems to be working. Apparently, rust provides some functions in some cases.